### PR TITLE
Issue/save unsave toggle ajax

### PR DESCRIPTION
**Pull request by vasanthakumar-a**

### CODE REVIEW

File: app/javascript/includes/save_post.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  $(".save").on("click", function(){
-    let post_id = $(this).data("id");
+  $(".saved").on("click", '.save', function(){
+    let post_id = $(this).parent().data("id");
 
     $.ajax({
-      url: "p/"+post_id+"/save",
-      method: "PATCH",
-      data: {},
+      url: "save_post?post_id="+post_id,
```
Comment (iMagesh) [inline app/javascript/includes/save_post.js:6]: why don't you use a clean url like the following:
```
/save-post/<post-id>
```

**Review by iMagesh — CHANGES_REQUESTED** [review]
Check comments

File: app/javascript/includes/save_post.js (hunk 1 above)
Comment (vasanthakumar-a) [reply app/javascript/includes/save_post.js]: url changed as '/save_post/post_id'

File: public/packs/manifest.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
```
[the remark this answers is not in the document]
Comment (vasanthakumar-a) [reply public/packs/manifest.json:1]: manifest flie added in gitignore

**Comment by vasanthakumar-a** [comment]
comments are resolved

**Pull request merged**